pkg/didcomm/packer/legacy/authcrypt/authcrypt_test.go

```diff
@@ -698,7 +703,8 @@ func Test_getCEK(t *testing.T) {
 		},
 	}
 
-	_, _, err := getCEK(recs, &k)
+	// nolint: dogsled
```

Comment:
perhaps a struct return would help?

Reply:
Made a hidden struct for these return values.